Frameworks/UIKit/UICTFont.mm

```diff
@@ -24,6 +24,19 @@ @implementation UIFontPrototype
 
 PROTOTYPE_CLASS_REQUIRED_IMPLS(UICTFont)
 
+// WinObjC-only extension for UINibUnarchiver
+- (instancetype)initWithCoder:(NSCoder*)coder {
+    NSString* name = [coder decodeObjectForKey:@"UIFontName"];
+    if ([name length] < 1) {
+        // fallback to default if could not find a font name
+        name = (__bridge NSString*)kCTFontDefaultFontName;
+    }
+
+    CGFloat size = [coder decodeFloatForKey:@"UIFontPointSize"];
+
+    return reinterpret_cast<UIFontPrototype*>(static_cast<UIFont*>(CTFontCreateWithName((__bridge CFStringRef)name, size, nullptr)));
+}
```

Comment:
- re: valuable for derived: this is a WinObjC-only extension, and _we_ don't have any UIFont derived classes, nor should we ever.
- re: designated initializers: UIFont has no initializers we could route to :(

---
In reply to: [85619643](https://github.com/Microsoft/WinObjC/pull/1263#discussion_r85619643) [](ancestors = 85619643)